engine/src/main/java/org/terasology/rendering/logic/LightComponent.java

```diff
@@ -43,14 +44,29 @@
     @Replicate
     public float lightAmbientIntensity = 1.0f;
 
+    /**
+     * This helps control how focused the specular light is. A smaller number will make a wider cone of light. A larger number will make a narrower cone of light.
+     */
     @Replicate
-    public float lightSpecularPower = 1.0f;
+    public float lightSpecularPower = 100.0f;
+    /**
+     * Light attenuation range used in the calculation of how light fades from the light source as it gets farther away.  It is use in the following calculation:
+     * <p>
+     * attenuation = 1 / (lightDist/lightAttenuationRange + 1)^2
+     * <p>
+     * Where lightDist is how far the point in the world is from the light source.
+     */
     @Replicate
-    public float lightAttenuationRange = 16.0f;
+    public float lightAttenuationRange = 10.0f;
```

Comment:
This range (and with the a 1.25 falloff value), will make a point light source act in roughly the same radius as a placed light.

Before (placed torch):
![terasology-160406170308-1280x720](https://cloud.githubusercontent.com/assets/5922109/14396044/3bd14340-fd8a-11e5-9370-68c88e10f1f9.jpg)

Before (held torch):
![terasology-160406170314-1280x720](https://cloud.githubusercontent.com/assets/5922109/14396052/41aa5f0e-fd8a-11e5-8de1-569a1c220161.jpg)

After (placed torch):
![terasology-160408093441-1280x720](https://cloud.githubusercontent.com/assets/5922109/14396062/485541e8-fd8a-11e5-9b6a-cacf50ddea4f.jpg)

After (held torch):
![terasology-160408093448-1280x720](https://cloud.githubusercontent.com/assets/5922109/14396072/50065544-fd8a-11e5-868a-7a0e1437743f.jpg)